src/databases.h

```diff
@@ -191,6 +191,7 @@ const Package_DB package_intel[] =
 	{ NULL,                             "Intel(R) Core(TM)2 Duo CPU E7400", "LGA 775"        },
 	{ NULL,                             "Intel(R) Core(TM)2 Quad CPU Q6600","LGA 775"        },
 	{ NULL,                             "Intel(R) Core(TM) i5-2500K CPU",   "LGA 1155"       },
+	{ "Sandy Bridge (Celeron)",	    "Intel(R) Celeron(R) CPU G550T",	"LGA 1155"	 },
```

Comment:
Both Codename and Model fields are not required at the same time.

Reply:
i see will remove it